core/src/main/scala/com/microsoft/azure/synapse/ml/codegen/CodeGen.scala

```diff
@@ -38,9 +38,12 @@ object CodeGen {
 
   def generateRClasses(conf: CodegenConfig): Unit = {
     val instantiatedClasses = instantiateServices[RWrappable](conf.jarName)
-    instantiatedClasses.foreach { w =>
-      println(w.getClass.getName)
-      w.makeRFile(conf)
+    // We omit deprecated LIME classes due to a conflict with the non-deprecated versions
```

Comment:
>// We omit deprecated LIME classes due to a conflict with the non-deprecated versions [](http://example.com/codeflow?start=4&length=85)

@mhamilton723 - we talked about this a while back. I can't remember what the conclusion was. This was the only way I could get the testgen on the explainer versions to work.

Reply:
remove the RWrappable (Comes in through Wrappable) trait from the classes and the RFuzzingTest trait from the corresponding tests

Reply:
The Wrappable trait comes in at LIMEBase, which is the base class for both the deprecated classes and their replacements, so removing RWrappable would involve some refactoring. Not sure why we'd want to remove that anyway, since don't we want the R wrapper to still be generated? 

For the test side, the deprecated LIMESuite classes inherit from EsitmatorFuzzing and TransformerFuzzing. Removing RTestFuzzing from those was my initial attempt, which I've removed. If this is the direction we want, I'll probably need a bit more guidance. 

For now, I'm now getting the correct results just by filtering in RTestGen. Seems a lot simpler than trait refactoring.